.travis.yml

```diff
@@ -20,14 +20,16 @@ env:
       - OS=fedora DIST=26
       - OS=fedora DIST=27
 
-install:
-  - sudo apt-get install coreutils
-
 script:
+  # lulpeg tags have the following format: major.minor
+  # They are not annotated and so can not be used by packpack
+  # automaticaly. Also packpack excpects major.minor.patch tag format.
```

Comment:
excpects → expects

Reply:
> packpack expects major.minor.patch tag format.

Packpack expects tags to be annotated and have major.minor.patch format or alternativally VERSION and RELEASE are set to major.minor.patch and commits count above the last tag.

Maybe such description would be more clear.

Reply:
Comment is fixed now.